cioos_data_transform/cioos_data_transform/utils/odf.py

```diff
@@ -14,6 +15,17 @@
 """
 
 
+def odf_dtypes(input=None):
+    """Return a dictionary with the mapping of the odf types to python types"""
+    odf_mapping = {'DOUB': 'float64', 'SING': 'float32', 'SYTM': 'float64', 'INTE': 'int32'}
+    if input and input in odf_dtypes():
```

Comment:
this line is confusing - couldnt you just say `if input and input in odf_mapping`?

Reply:
Just converted that function to a simple dictionary